Applications/Spire/Source/UiViewer/MarketComboBoxTestWidget.cpp

```diff
@@ -0,0 +1,26 @@
+#include "Spire/UiViewer/MarketComboBoxTestWidget.hpp"
+#include <QHBoxLayout>
+#include "Nexus/Definitions/DefaultMarketDatabase.hpp"
+#include "Spire/Ui/MarketComboBox.hpp"
+#include "Spire/Ui/CustomQtVariants.hpp"
+
+using namespace boost::signals2;
+using namespace Nexus;
+using namespace Spire;
+
+MarketComboBoxTestWidget::MarketComboBoxTestWidget(QWidget* parent)
+    : ComboBoxAdapter(parent) {
+  auto layout = new QHBoxLayout(this);
+  layout->setContentsMargins({});
+  auto database = GetDefaultMarketDatabase();
```

Comment:
Inline the expression directly unless you intend to use the expression multiple times.